src/test/java/com/google/sps/comments/CommentTest.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runners.JUnit4;
+import org.junit.runner.RunWith;
+
+import java.lang.IllegalArgumentException;
+
+import com.google.sps.data.Comment;
+
+/*The Comment class has the following fields
+ *  new Comment(long id, String comment, long timestamp)
+ */
+@RunWith(JUnit4.class)
+public final class CommentTest {
+    private static final int NEW_ID = 123;
+    private static final String NEW_COMMENT = "comment";
+    private static final int NEW_TIMESTAMP = 456;
+    private static final Comment comment = new Comment(NEW_ID, NEW_COMMENT, NEW_TIMESTAMP);
```

Comment:
it makes more sense to create this within the methods (`testGetComment()` and `testGetTimestamp()`). Usually a test should be self contained and since the result in your test depends on what you pass in arguments to the constructor here, creating the `Comment` object each time would be the better way to go about it.

Reply:
Oh ok, sounds good!